src/app/Http/Controllers/QuizController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class QuizController extends Controller
+{
+    public function index1() {
+    return view('quiz1');
+    }
```

Comment:
フォーマットすると直ると思いますが メソッド名のあとは改行してあると統一されて可読性あがるので良いです
```suggestion
    public function index1() 
    {
        return view('quiz1');
    }
```